filetests/isa/intel/baseline_clz_ctz_popcount.cton

```diff
@@ -0,0 +1,51 @@
+
+test compile
+set is_64bit
+isa intel baseline
+
+
+; clz/ctz on 64 bit operands
+
+function %i64_clz(i64) -> i64 {
+ebb0(v10: i64):
+  v11 = clz v10
+  ; check: x86_bsr
+  ; check: selectif.i64
+  return v11
+}
+
+function %i64_ctz(i64) -> i64 {
+ebb1(v20: i64):
+  v21 = ctz v20
+  ; check: x86_bsf
+  ; check: selectif.i64
+  return v21
+}
+
+
+; clz/ctz on 32 bit operands
+
+function %i32_clz(i32) -> i32 {
+ebb0(v10: i32):
+  v11 = clz v10
+  ; check: x86_bsr
+  ; check: selectif.i32
+  return v11
+}
+
+function %i32_ctz(i32) -> i32 {
+ebb1(v20: i32):
+  v21 = ctz v20
+  ; check: x86_bsf
+  ; check: selectif.i32
+  return v21
+}
+
+
+; popcount on 64 bit operands
+
+function %i64_popcount(i64) -> i64 {
+ebb0(v30: i64):
+  v31 = popcnt v30;
```

Comment:
This test is lacking `check:` lines. Also, should there be an `i32_popcount` test?